content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
+updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET). The API includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. Kubernetes also supports efficient change notifications on resources via "watches" and consistent lists, so that API clients can effectively cache, track, and synchronize the state of resources.
+
+## Kubernetes API terminology {#standard-api-terminology}
+
+Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, such as a {{< glossary_tooltip term_id="pod" >}} or {{< glossary_tooltip term_id="namespace" >}}. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of kind `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects have a unique [names](/docs/concepts/overview/working-with-objects/names/#names) to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
 
-## Standard API terminology
+Kubernetes relies on terminology from [ReST](https://en.wikipedia.org/wiki/Representational_state_transfer)
+to name and describe several API concepts:
 
-Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, like a pod or namespace. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects will have a unique name to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
+* A **resource type** is the name used in the URL (for example: `pods`, `namespaces`, `services`);
+* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**;
+* A set of instances of a resource type is known as a **collection**;
+* A single instance of the resource type is called an **object**, or sometimes a **resource**.
 
-Kubernetes generally leverages standard RESTful terminology to describe the API concepts:
+{{< note >}}
+In URLs, Kubernetes types such as `pod` and `configmap` are **lowercase**. This is different from
```

Comment:
This sentence is polluting the who section. If we are talking about "Kubernetes kinds", we should say `Pod` and `ConfigMap`, if we are instead talking about types, we use their lower cased plurals `pods` and `configmaps`.

Reply:
Let's try to find another way that ensures that readers know why the case in the API is different from the rest of the documentation.

Reply:
Is it important to mention capitalization conventions in this reference?
I would hope that readers interpret the object specification correctly. I would not question a lowercase term in a URL.

Reply:
> Is it important to mention capitalization conventions in this reference?

I think so - I'd been using Kubernetes for quite a while before I understood _why_ RBAC uses lowercase even though tools like `kubectl` print similar-looking names in UpperCamelCase. I'd like there to be at least one place in the documentation where someone can find that answered.

Reply:
This revision is wrong. In URLs, we only have lowercase plurals for resources.